life/updated/02-life.adoc

```diff
@@ -1568,447 +1566,444 @@ for we were all committed to prison.
+These put by the jury that were chosen of our neighbors,
 and called another jury presently in the court,
 such as they thought most fit for their turn.
-Then the clerk began, and read an indictment, namely, "`That we were found,
+Then the clerk began, and read an indictment, namely, that we were found,
 or taken at an unlawful assembly, with force of arms,
 in contempt of the king and his laws, crown and dignity,
-to the terror of the people,`" etc.
-And he said to the jury, "`Gentlemen, you have heard their indictment,
-if you find them guilty, you find for the king.`"
+to the terror of the people, etc.
+And he said to the jury, "`Gentlemen, you have heard their indictment.
+If you find them guilty, you find for the king.`"
 And a bishop who sat upon the bench with the judge, stood up and said,
-"`That the first Quaker that ever was in England,
+"`The first Quaker that ever was in England
 was hanged for being concerned in the Popish plot.`"
-I answered, that the first who was called a Quaker, was now alive.
+I answered that the first who was called a Quaker was still alive.
 He said again,
-"`He could prove by sufficient witness that he
+"`I can prove by sufficient witness that he
 was hanged for being one in the Popish plot.`"
 Then the bishop being enraged, because he was contradicted, held up his hand towards us,
-and bid us "`Have a care what we said, for those who had estates among us,
-it should cost them their estates,
-and they that had not should lie in prison until they perished.`"
-Such was their rage and wickedness against us, that it was very grievous to hear them;
-but there was a secret cry many times ran through my heart unto the Lord, "`Lord,
-work for your name`'s sake, and confound their wisdom, and rage,
-and bring down their proud and wicked spirits,
-and bring to naught their mischievous contrivance,
-that they have been contriving against your innocent people,
-as they have been making themselves merry, and drinking wine to the full,
+and bid us have a care what we said, for to those who had estates among us,
+it would cost them their estates,
+and they that had none would lie in prison until they perished.`"
+Such was their rage and wickedness against us, that it was very grievous to hear them,
+but this was a secret cry many times which ran through my heart unto the Lord, "`Lord,
+work for Your name`'s sake, and confound their wisdom, and rage,
+and bring down their proud and wicked spirits.
+Bring to naught their mischievous contrivance
+that they have been contriving against Your innocent people,
+as they have been making themselves merry and drinking wine to the full,
 and feeding themselves with the fatness of the earth, as the rich man did,
-and have what their hearts lust after,
-and yet none of all these things will give them content nor satisfaction,
+and have what their hearts lust after.
+And yet none of all these things will give them content nor satisfaction
```

Comment:
I think this reads a little better with a comma instead of the period. And also change content for contentment. Like this: and have what their hearts lust after, and yet none of all these things will give them contentment or satisfaction...